### PR TITLE
[frontend] Exclusion List frontend Create - Delete (#8941)

### PR DESCRIPTION
**Pull request by frapuks**
### Proposed changes

* Add Exclusion Lists page on Stteing / Customization
* Creation form with a file
* Deletion
* Enable / disable

### Related issues

* #8941 

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

Next chunk for: 
- Creation form with content
- Edition form
- Sync status

### CODE REVIEW

File: opencti-platform/opencti-front/lang/front/en.json [hunk 1]
```diff
@@ -973,6 +976,7 @@
   "Entity tags": "Entity tags",
   "Entity type": "Entity type",
   "Entity types": "Entity types",
+  "Entity Types": "Entity Types",
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/lang/front/en.json]: why not use directly "Entity types" that we already have in our translations ?

File: opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListPopover.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        <MenuItem onClick={handleDisplayEdit} disabled>{t_i18n('Update')}</MenuItem>
+        <MenuItem onClick={handleOpenDelete}>{t_i18n('Delete')}</MenuItem>
+      </Menu>
+      <DeleteDialog
+        title={t_i18n('Do you want to delete this exclusion list?')}
+        deletion={deletion}
+        submitDelete={submitDelete}
+      />
+      <Drawer
+        title={t_i18n('Exclusion list edition')}
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListPopover.tsx]: We usually use a title like "Update a ..." same as "Create an exclusion list"=> "Update an exclusion list"

File: opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListEdition.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  const initialValues: ExclusionListEditionValues = {
+    name: data?.name ?? '',
+    description: data?.description,
+    exclusion_list_entity_types: (data?.exclusion_list_entity_types ?? []).map((type) => ({
+      value: type,
+      label: type,
+    })),
+  };
+
+  const entityTypes: ExclusionListEntityTypes[] = ['IPV4_ADDR', 'IPV6_ADDR', 'DOMAIN_NAME', 'URL'];
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListEdition.tsx]: we should list all possible stix cyber observable entity types, not this static list, which doesn't even correspond to a valid entity type

Comment (SouadHadjiat) [reply opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListEdition.tsx]: you should be able to get them like this : 
```
const { schema: { scos: entityTypes } } = useSchema();
```

File: opencti-platform/opencti-front/lang/front/en.json (hunk 1 above)
Comment (frapuks) [reply opencti-platform/opencti-front/lang/front/en.json]: Done

File: opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListPopover.tsx (hunk 2 above)
Comment (frapuks) [reply opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListPopover.tsx]: Done

File: opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListEdition.tsx (hunk 3 above)
Comment (frapuks) [reply opencti-platform/opencti-front/src/private/components/settings/exclusion_lists/ExclusionListEdition.tsx]: Done. We need to verify with the product about this list, and if it's really able to create exclusion list for all observable types. Valentin is on it.

File: opencti-platform/opencti-front/src/private/components/settings/CustomizationMenu.tsx [hunk 4]
```diff
@@ -23,6 +23,10 @@ const CustomizationMenu: FunctionComponent = () => {
       path: '/dashboard/settings/customization/decay',
       label: 'Decay rules',
     },
+    {
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/settings/CustomizationMenu.tsx]: add feature flag

File: opencti-platform/opencti-front/src/private/components/settings/Root.jsx [hunk 5]
```diff
@@ -328,6 +329,14 @@ const Root = () => {
               </Security>
             }
           />
+          <Route
```
Comment (SouadHadjiat) [inline opencti-platform/opencti-front/src/private/components/settings/Root.jsx]: add feature flag

**Review by JeremyCloarec — APPROVED** [review]

**Pull request merged**